.annotations

```diff
@@ -0,0 +1,15 @@
+source_path: ../
```

Comment:
Should this be `../` or `./`?  wouldn't `../` search other files outside the repo?

Reply:
Good catch! It doesn't affect this CI, since the Docker container does this checking from this directory:
`/edx/app/edx_notes_api/edx_notes_api`
But the checking *should* be done from `./` - will fix.